dataPipelines/gc_scrapy/gc_scrapy/pipelines.py

```diff
@@ -119,7 +119,7 @@ def get_media_requests(self, item, info):
         file_item = self.get_first_supported_downloadable_item(item["downloadable_items"])
 
         if file_item:
-            url = file_item["web_url"]
+            url = file_item["download_url"]
```

Comment:
we need to rigorously check that changing this variable doesn't break our pipeline. this will need to get addressed in the CAC crawlers too